monai/data/image_writer.py

```diff
@@ -106,17 +107,23 @@ def resolve_writer(ext_name, error_if_not_found=True) -> Sequence:
     if fmt.startswith("."):
         fmt = fmt[1:]
     avail_writers = []
+    required_pkg = []
     default_writers = SUPPORTED_WRITERS.get(EXT_WILDCARD, ())
     for _writer in look_up_option(fmt, SUPPORTED_WRITERS, default=default_writers):
         try:
             _writer()  # this triggers `monai.utils.module.require_pkg` to check the system availability
             avail_writers.append(_writer)
-        except OptionalImportError:
+        except OptionalImportError as e:
+            error_match = re.search(r"`(.*?)`", str(e))
+            if error_match:
+                required_pkg.append(error_match.group(1))
             continue
         except Exception:  # other writer init errors indicating it exists
             avail_writers.append(_writer)
     if not avail_writers and error_if_not_found:
-        raise OptionalImportError(f"No ImageWriter backend found for {fmt}.")
+        raise OptionalImportError(
```

Comment:
Could you please also include the test case in `test_image_rw.py` to ensure the change works as expected? Thanks.

Reply:
@KumoLiu, should it be added as a separate class?